bullet/client.py

```diff
@@ -197,7 +205,11 @@ def moveDown(self):
     def accept(self):
         utils.moveCursorDown(len(self.choices) - self.pos)
         cursor.show_cursor()
-        ret = self.choices[self.pos]
+        #flag check and return ether objects or just choices
+        if(self.objects):
```

Comment:
Nitpicking, but you generally leave off parentheses on if statements in Python for simple conditions like this.

Also, try to be consistent with whitespace to match the surrounding code.